### PR TITLE
fix use of pluto change files

### PR DESCRIPTION
**Pull request by damonmcc**
## changes
* generalize approach to getting pluto change zip file and csv files to accommodate filename changes introduced in https://github.com/NYCPlanning/db-pluto/pull/425
* preserve backwards compatibility by always looking for both new and old sets of filename
* use `pluto_changes_applied.csv` for `Review Manual Corrections` section of PLUTO QAQC page 
* simplify uses of `DigitalOceanClient` in all `helpers.py` files
* add an `example.env` file
* impore intro section of PLUTO page
  * remove badges from PLUTO page (they already exist in PLUTO repo readme and can be misleading)
  * incorporate text changes from Te's PR https://github.com/NYCPlanning/data-engineering-qaqc/pull/253
* update readme dev details

### CODE REVIEW

File: src/pluto/helpers.py [hunk 1]
```diff
@@ -46,24 +50,47 @@ def get_data(branch) -> Dict[str, pd.DataFrame]:
...
-    pluto_corrections_zip = client.zip_from_DO(
-        zip_filename=f"db-pluto/{branch}/latest/output/pluto_corrections.zip",
-    )
-
-    rv["pluto_corrections"] = client.unzip_csv(
-        csv_filename="pluto_corrections.csv", zipfile=pluto_corrections_zip
+    valid_changes_files_group = [
+        {
+            "zip_filename": "pluto_changes.zip",
+            "applied_filename": "pluto_chanes_applied.csv",
```
Comment (mbh329) [inline src/pluto/helpers.py:58]: should be `pluto_changes`

**Review by mbh329 — APPROVED** [review]
Looks good to me - nice removing duplicative code in the repo level python files

File: src/digital_ocean_client.py [hunk 2]
```diff
@@ -39,14 +39,20 @@ def s3_resource(self):
             endpoint_url=os.getenv("AWS_S3_ENDPOINT"),
         )
 
-    def get_all_folders_in_repo(self):
+    def get_all_folder_names_in_repo_folder(self):
```
Comment (mbh329) [inline src/digital_ocean_client.py:42]: Function names are much more detailed...NOICE

**Comment by damonmcc** [comment]
made and tested replacing `correction` with `changes` in both text and code

**Pull request merged**